### PR TITLE
Release v0.14.0

### PR DESCRIPTION
**Pull request by sjcobb**
- bump versions to v0.14.0, update changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 # Changelog
 
-## Unreleased
+## 0.14.0 / 2022-11-2
```
Comment (Nexucis) [inline CHANGELOG.md:3]: I think the script failed more because of the date format @sjcobb .

```suggestion
## 0.14.0 / 2022-11-02
```

Comment (sjcobb) [reply CHANGELOG.md]: > I think the script failed more because of the date format @sjcobb .

😆 oh dang, thanks!

**Review by LukeTillman — APPROVED** [review]
🤞

**Review by Nexucis — APPROVED** [review]
looks good to me.
As a side note, perhaps we want to enforce the modification of the changelog in each PR that proposed something visible by the user ?

**Comment by LukeTillman** [comment]
> 

Yup, agreed. I didn't realize we'd started doing the Unreleased section yet, so I'll definitely start looking for it in PRs.

**Pull request merged**